### ISSUE TITLE
code error?

### ISSUE THREAD

**Issue by DesireHope** [zhijing-jin/GraphIE#1]
When i run python examples/multi_runs_conll.py --gpu_id 0, i get error:

---------------------------------------------------------------------- 
Reading data from data/dset/03co/train.c_w_d_dw_ds_sw_word_ibo_dic
[Info] Loading graph: ./data/run/../dset/03co/graph/train_graph.json
Traceback (most recent call last):
  File "examples/NERCRF_conll.py", line 548, in <module>
    main()
  File "examples/NERCRF_conll.py", line 240, in main
    cheat_densify=cheat_densify)
  File "./neuronlp2/io/conll03_data.py", line 146, in read_data
    inst = reader.getNext(unk_replace, normalize_digits, graph_model)
  File "./neuronlp2/io/reader.py", line 220, in getNext
    for sent in words])
  File "./neuronlp2/io/reader.py", line 220, in <listcomp>
    for sent in words])
  File "/home/ubuntu/anaconda3/envs/dp-py3-env/lib/python3.6/site-packages/numpy/lib/arraypad.py", line 1377, in pad
    newmat = _append_const(newmat, pad_after, after_val, axis)
  File "/home/ubuntu/anaconda3/envs/dp-py3-env/lib/python3.6/site-packages/numpy/lib/arraypad.py", line 142, in _append_const
    (arr, (np.zeros(padshape) + val).astype(arr.dtype)), axis=axis)
TypeError: ufunc 'add' did not contain a loop with signature matching types dtype('<U32') dtype('<U32') dtype('<U32')

**Comment by zhijing-jin** [comment]
DesireHope and I have solved this issue a while ago. (See our [solution](https://github.com/thomas0809/GraphIE/issues/2).)

**TL;DR: This is due to the version of numpy. Try updating your numpy version to `numpy 1.15.4` will avoid this error on dtype('<U32').**
